### ISSUE TITLE
 how to recognize the blank between two English words

### ISSUE THREAD

**Issue by cvchongci** [ouyanghuiyu/chineseocr_lite#131]
How to recognize blank between two English words?
for my current model, if I input one English sentence then the output will concatenate all the English words together. for example:
inputed image:
<img width="363" alt="36" src="https://user-images.githubusercontent.com/62248916/80300616-539c4c00-87af-11ea-93d8-1ba1b150594d.png">
recognized result:

> A------l-ll-t-h--e--r-e-c--o--g-n-i-tiio---n--a-c-cc--ur-a--c-i-e-s---o--n--t-h---e => Alltherecognitionaccuraciesonthe

So how to recognize the blank between two English words?

**Comment by Zrufy** [comment]
hi how do you solve the problem of the output model with error core dumped?

**Comment by cvchongci** [comment]
yes, please refer to https://github.com/Holmeyoung/crnn-pytorch/issues/48#issuecomment-620557523

**Comment by Zrufy** [comment]
I wanted to ask you if the model obtained with Holmeyoung / crnn-pytorch are you able to convert it to ncnn. Thanks for helping me.

**Comment by cvchongci** [comment]
I'm not sure. I have no experience in converting model to ncnn.
